androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/playerOverlay/TimeBarComponent.kt

```diff
@@ -5,19 +5,24 @@ import androidx.compose.foundation.layout.Box
 import androidx.compose.material3.Slider
 import androidx.compose.material3.SliderDefaults
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
 import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
 import androidx.compose.ui.graphics.Color
 
 @Composable
 fun TimeBarComponent(
     currentTime: () -> Long,
     bufferedPosition: () -> Long,
     duration: Long,
-    onTimeChange: (Long) -> Unit,
-    onTimeChangeFinished: () -> Unit,
+    onTimeChangeFinished: (Long) -> Unit,
     interactionSource: MutableInteractionSource
 ) {
     val valueRange = remember { 0f..duration.toFloat().coerceIn(1f, Float.MAX_VALUE) }
```

Comment:
I know that isn't related to you PR, but it is good to add a key on this `remember`. Like:
```
val valueRange = remember(duration) { 0f..duration.toFloat().coerceIn(1f, Float.MAX_VALUE) }
```